### ISSUE TITLE
Site: View Admin should indicate launch in Safari

### ISSUE THREAD

**Issue by astralbodies** [wordpress-mobile/WordPress-iOS#958]
The row labeled "View Admin" should not have a disclosure accessory but rather some distinction that it will exit the app to open.

**Comment by astralbodies** [comment]
Still an issue in 6.2. @mattmiklic what do you suggest?

**Comment by mattmiklic** [comment]
Here's what I'd recommend. There's a standard icon metaphor for "external link" in iOS and we have a version of it in gridicons (`gridicons-external`). I'd recommend using that icon in `greyLighten20` at 12px (not a standard size for Gridicons, but I've tested it at this size, and it will work well). At 12px, `gridicons-external` is a good match for the standard disclosure indicator chevron.

![external-link](https://cloud.githubusercontent.com/assets/1396552/14749126/abd5f17a-088b-11e6-965f-447fb2144ead.png)

Speaking of that chevron -- I just noticed that it's #d7d7d7, the default color for iOS. It would be nice if we could customize those to `greyLighten20`, so they matched the rest of our color palette. I'm not sure if customize those icons would be a major undertaking so it might be something we consider separately.
